package.json

```diff
@@ -10,10 +10,12 @@
     "url": "https://github.com/taskcluster/taskcluster-login.git"
   },
   "scripts": {
-    "compile": "babel-compile -c taskcluster-lib-rules/babel src:lib",
+    "compile": "babel-compile -c taskcluster-lib-rules/babel src:lib test:.test",
```

Comment:
I think to be Truly Modern™ we should be using `babel-compile -p taskcluster ...` with `babel-compile@2.x` and `babel-preset-taskcluster`. Not necessary for this review though.